client/src/main/java/io/pravega/client/stream/impl/StreamCutImpl.java

```diff
@@ -74,6 +87,50 @@ public String toString() {
                                                                                e -> e.getValue())));
     }
 
+    @Override
+    public String asText() {
+        return compressToBase64(getText());
```

Comment:
Say we change the method we use to generate a textual representation in the future, how do we make the reconstruction compatible? Or do we simply throw an error at the application?

Reply:
`StreamCutImpl#TO_STRING_VERSION` can be used to handle multiple versions of textual representation.